hpke.go

```diff
@@ -256,12 +320,29 @@ func (ctx *EncryptContext) Seal(aad, pt []byte) []byte {
 	return ct
 }
 
+func (ctx *EncryptContext) Marshal() ([]byte, error) {
```

Comment:
This can just go on `cipherContext`, and you can still call it on `EncryptContext` and `DecryptContext`.

Reply:
Done. (Similarly for `DecryptContext`.)